source/air/src/printer.rs

```diff
@@ -236,6 +236,12 @@ impl Printer {
                             nodes.push(str_to_node(":pattern"));
                             nodes.push(self.exprs_to_node(trigger));
                         }
+                        if let Some(s) = qid {
+                            nodes.push(str_to_node(":qid"));
+                            nodes.push(str_to_node(s));
+                            nodes.push(str_to_node(":skolemid"));
+                            nodes.push(str_to_node(format!("skolem_{}", s).as_str()));
```

Comment:
Can you please add `"skolem"` to `def.rs`? And ideally check that `skolem_{}` appears correctly in the input in `parser.rs`?

Reply:
I added `"skolem"` to `def.rs`, but I'm not sure what you have in mind for `parser.rs`.  My understanding was that `parser.rs` should be able to handle arbitrary AIR files, which could be handwritten or generated by something other than Verus, in which case we can't necessarily expect someone to use the same convention for skloem identifiers.  Apologies if I've misunderstood the suggestion though!

Reply:
The AIR file format is pretty much currently defined by the printer and parser. Without a change to `parser.rs` an AIR file with a skolem id that doesn't match `skolem_{:qid}` would not be parsed correctly (`printer` accepts the tokens but then throws it away): I'd rather we rejected AIR that didn't match the only format AIR can currently emit and represent in its AST; otherwise if someone hand-wrote some other skolemid, it would be silently ignored, which would be confusing to debug. In fact: I'd say we should only accept AIR where either both :qid and :skolemid are present or both are absent.
Note that printer (in this case) also determines what the SMT output is.

In other words: something that would be accepted by the AIR parser right now wouldn't correctly round-trip when parsed and then printed (either as AIR or SMT).

Let me know if I've misunderstood, of this is unclear.

Reply:
I see; thanks for the explanation.  I've pushed a fix.